azure-pipeline-templates/build-release.yml

```diff
@@ -44,6 +44,22 @@ steps:
     displayName: "Test Binary built"
     continueOnError: false
 
+  # Build the health-monitor binary  
+  - task: Go@0
+    inputs:
+      command: 'build'
+      arguments: "-o healthmon ./tools/health-monitor/"
+      workingDirectory: ${{ parameters.work_dir }}
+    displayName: 'Building health-monitor binary'
+  
+  # Verifying whether built binary is correct
+  - script: |
+      sudo chmod +x ./healthmon
+      ./healthmon --version
```

Comment:
We need to package this binary as well by copying it to respective folder. Don't see that step in release pipeline.